sidecar/supervisor/appsupervisor.go

```diff
@@ -173,7 +195,7 @@ func terminateSubprocesses(procs []*process, sig os.Signal) {
 		if time.Now().After(timeout) {
 			for _, proc := range procs {
 				if proc.Cmd.Process != nil {
-					proc.Cmd.Process.Kill()
+					syscall.Kill(-proc.Cmd.Process.Pid, syscall.SIGKILL)
```

Comment:
Two things
1) Do we always want to kill the group in this case, regardless of the KillGroup setting?
2) Is `syscall.Kill(...)` synchronous? If so, could we speed this up with a wait group?